analyzer/tools/build-logger/src/ldlogger-util.c

```diff
@@ -708,5 +706,5 @@ int logPrint(char* logLevel_, char* fileName_, int line_, char *fmt_,...)
   fclose(stream);
   freeLock(lockFd);
 
-  return 0;
+  return;
```

Comment:
There is no need for this return statement here:
```suggestion
```

Reply:
I'm not even sure how I missed that xD